presentations/tagging-provider.md

```diff
@@ -0,0 +1,722 @@
+# Tagging Provider
```

Comment:
use plural of tagging provider (tagging providers) instead 
also replace other usages